browser/resources/settings/brave_wallet_page/brave_wallet_page.ts

```diff
@@ -233,6 +233,10 @@ class SettingsBraveWalletPage extends SettingsBraveWalletPageBase {
   onNftDiscoveryEnabledChange_() {
     this.browserProxy_.setNftDiscoveryEnabled(this.$.enableNftDiscovery.checked)
   }
+
+  onPrivateWindowsEnabledChange_() {
+    this.browserProxy_.setPrivateWindowsEnabled(this.$.enableNftDiscovery.checked)
```

Comment:
I don't think this is correct, it's not used either, please remove this change. Same for NftDiscovery one above.